<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>832</width>
    <height>588</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SendMessageSafe（安心傳訊）</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="confirmBtn">
    <property name="geometry">
     <rect>
      <x>632</x>
      <y>70</y>
      <width>121</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string>Confirm</string>
    </property>
   </widget>
   <widget class="QLabel" name="vi_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>AES VI:</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="vi_input">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>40</y>
      <width>461</width>
      <height>71</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;01020304050607080&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="encryptBtn">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>240</y>
      <width>93</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Encrypted</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="msg_input">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>243</y>
      <width>461</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="decryptBtn">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>240</y>
      <width>93</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Decrypted</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="output">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>333</y>
      <width>711</width>
      <height>181</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="msg_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>Message:</string>
    </property>
   </widget>
   <widget class="QLabel" name="output_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>Output:</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="key_input">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>140</y>
      <width>461</width>
      <height>71</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;hhhhhhhhhhhhhhhh&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="key_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>AES Key:</string>
    </property>
   </widget>
   <widget class="QLabel" name="about_label">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>560</y>
      <width>431</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Marco Cheung @ 2022, https://github.com/marc0cheung/</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>530</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Screen Decryption</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
